src/ARMeilleure/Instructions/InstEmitSimdCvt32.cs

```diff
@@ -225,24 +225,14 @@ private static Operand EmitRoundMathCall(ArmEmitterContext context, MidpointRoun
 
         private static FPRoundingMode RMToRoundMode(int rm)
         {
-            FPRoundingMode roundMode;
-            switch (rm)
-            {
-                case 0b00:
-                    roundMode = FPRoundingMode.ToNearestAway;
-                    break;
-                case 0b01:
-                    roundMode = FPRoundingMode.ToNearest;
-                    break;
-                case 0b10:
-                    roundMode = FPRoundingMode.TowardsPlusInfinity;
-                    break;
-                case 0b11:
-                    roundMode = FPRoundingMode.TowardsMinusInfinity;
-                    break;
-                default:
-                    throw new ArgumentOutOfRangeException(nameof(rm));
-            }
+            var roundMode = rm switch
```

Comment:
```suggestion
            return rm switch
```
Can't it just do this directly?